### DISCUSSION TITLE
Welcome to community Discussions!

### DISCUSSION THREAD

**Discussion by bluejayA** [openinfradev/community#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by intelliguy** [comment]
HANU는 한글만 쓰기로 한거 아닌가요?

**Reply by bluejayA** [reply]
이건 discussion 채널 만들면 github가 자동으로 만들어주는..... (을 알지만 농담을 하신건가?)
